### PR TITLE
DT-1284: 🎨 switch userdetails from react to nunjucks

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: sass/_govuk-overrides.scss [hunk 1]
```diff
@@ -1 +1,6 @@
 // Whenever we have/want to diverge from govuk-frontend
+.user-enabled-status {
+  .govuk-table__cell {
+    border-bottom: 0
```
Comment (petergphillips) [inline sass/_govuk-overrides.scss:4]: needed as first table doesn't want a bottom border according to design

File: src/Admin/MaintainAuthUsers/containers/AuthUserCreateContainer.test.js [hunk 2]
```diff
@@ -132,7 +132,6 @@ describe('Auth create container', () => {
         { email: 'user@created.com', firstName: 'first', lastName: 'last', username: 'userme', groups },
         { params: { username: 'userme' } }
       )
-      expect(mockHistory.push).toBeCalledWith('/maintain-auth-users/userme')
```
Comment (petergphillips) [inline src/Admin/MaintainAuthUsers/containers/AuthUserCreateContainer.test.js:135]: removed as will be converting all shortly anyway

File: src/Admin/MaintainAuthUsers/containers/AuthUserSearchResultsContainer.test.js [hunk 3]
```diff
@@ -55,54 +55,4 @@ describe('Auth search results container', () => {
       expect(submitEvent.preventDefault).toBeCalled()
     })
   })
-
-  describe('handle edit', () => {
```
Comment (petergphillips) [inline src/Admin/MaintainAuthUsers/containers/AuthUserSearchResultsContainer.test.js:59]: removed as will be converting all anyway

**Review by SimonMitchellMOJ — APPROVED** [review]
LGTM

**Pull request merged**